### PR TITLE
Sports Betting - My Account - Page Layout

### PR DESCRIPTION
**Pull request by rubyweb3**
https://github.com/AugurProject/augur/issues/6977

Screen Shot 2020-04-09 at 6 58 07 PM
Screen Shot 2020-04-09 at 6 57 55 PM
Screen Shot 2020-04-09 at 6 57 42 PM

### CODE REVIEW

File: packages/augur-ui/src/assets/styles/_typography.less [hunk 1]
```diff
@@ -192,13 +193,18 @@
 @size-326: 20.375rem; // 326px
 @size-340: 21.25rem; // 340px
 @size-350: 21.875rem; // 350px
+@size-360: 22.5rem; // 360px
 @size-400: 25rem; // 400px
 @size-480: 30rem; // 480px
+@size-550: 34.375rem; // 550px
 @size-560: 35rem; // 560px
+@size-600: // 600px
```
Comment (JohnDanz) [inline packages/augur-ui/src/assets/styles/_typography.less]: missing a value here

File: packages/augur-ui/src/modules/account/components/account-view.styles.less [hunk 2]
```diff
@@ -140,10 +140,33 @@
...
+      margin: @size-32;
+    }
+  }
+
   .AccountView {
+    margin: @size-8 @size-32;
+    display: flex;
+    flex-flow: column wrap;
+    align-content: space-between;
+    min-height: @size-1400;
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/account/components/account-view.styles.less]: I would prefer to use grid layout for this page, flex is workable but grid is preferred for layouts that are more complex like this one.

File: packages/augur-ui/src/modules/account/components/account-view.styles.less [hunk 3]
```diff
@@ -170,23 +191,50 @@
...
+    }
+
+    > footer {
+      min-height: @size-40;
     }
   }
+
+  /* Force new columns */
+  .AccountView::before,
+  .AccountView::after {
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/account/components/account-view.styles.less]: I would rather use grid to avoid things like this

**Review by JohnDanz — CHANGES_REQUESTED** [review]
Let's switch to grid for the layout, also remember we will be adding another list of items (Created Markets) so grid will allow us to more easily integrate a new component then trying to re-adjust with flex.

File: packages/augur-ui/src/assets/styles/_typography.less (hunk 1 above)
Comment (rubyweb3) [reply packages/augur-ui/src/assets/styles/_typography.less]: Updated.

File: packages/augur-ui/src/modules/account/components/account-view.styles.less (hunk 3 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/account/components/account-view.styles.less]: Removed.

File: packages/augur-ui/src/modules/account/components/account-view.styles.less (hunk 2 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/account/components/account-view.styles.less]: Updated to grid layout. Please let me know what you think about it. Thanks.

File: packages/augur-ui/src/modules/account/components/account-view.styles.less [hunk 4]
```diff
@@ -140,53 +140,85 @@
 
 :root[theme="BETTING"],
 :root[theme="SPORTS"] {
-  .AccountView {
-    > div {
-      border: @size-1 solid var(--color-outline);
-      border-radius: @border-radius-rounded;
+  .AccountViewWrap {
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/account/components/account-view.styles.less]: we shouldn't need `AccountViewWrap`, the HTML should change very little in `account-view.tsx` in terms of its structure. we would want ot just modify `AccountView`.

File: packages/augur-ui/src/modules/account/components/account-view.tsx [hunk 5]
```diff
@@ -113,6 +115,35 @@ export default class AccountView extends React.Component<
...
+                    <Overview hideHeader={false} />
+                    <AugurStatus hideHeader={false} />
+                  </div>
+                  <div>
+                    <Transactions />
+                    <Favorites />
+                  </div>
+                </div>
+                <TermsAndConditions />
+              </div>
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/account/components/account-view.tsx:134]: I'd simplify this to just a div with `AccountView` and everything inside of it with no added divs for row separation. Use grid to place the pieces where they belong.

**Review by JohnDanz — CHANGES_REQUESTED** [review]

File: packages/augur-ui/src/modules/account/components/account-view.styles.less (hunk 4 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/account/components/account-view.styles.less]: Removed.

File: packages/augur-ui/src/modules/account/components/account-view.tsx (hunk 5 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/account/components/account-view.tsx:134]: Updated. Some UIs have not been updated yet. I will work on it.

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**